media-video/aegisub/aegisub-9999.ebuild

```diff
@@ -94,12 +92,14 @@ src_configure() {
 		$(use_with pulseaudio libpulse)
 		$(use_with spell hunspell)
 		$(use_with uchardet)
+		--disable-compiler-flags
```

Comment:
Without this configure option, aegisub uses some outdated compiler option and overwrites CFLAGS, CXXFLAGS and such.